ironfish/src/consensus/verifier.ts

```diff
@@ -654,6 +654,41 @@ export class Verifier {
     // }
 
     // TODO(jwp): verify shielding/mints balance
+
+    const shieldBalance: BufferMap<bigint> = new BufferMap()
+    const mintBalance: BufferMap<bigint> = new BufferMap()
```

Comment:
we could also do this with a single `BufferMap` or `AssetBalances` instance if we kept track of deltas/balances for each asset